bin/gendocs

```diff
@@ -22,6 +22,12 @@ if [ -d venv ]; then
     source venv/bin/activate
 fi
 
+# Ensure git friendly files are generated by enforcing version
+if ! pip3 freeze | grep -q "json-schema-for-humans==0.39.1"; then
```

Comment:
this version should be explicitly specified in the etc/requirements.txt file -- I'm not sure a runtime check here is appropriate.  They do slightly different things, but generally speaking the requirements.txt way is the standard way so an explicit check isn't normal/expected.

Reply:
what's the utility of keeping this check here?   Usually we just rely on the requirements.txt enforcement b/c it keeps it uniform and simple. If there's some reason to have this explicit check here, then it should at least be clarified as to _why_ its there in the comment.  ("why" as in why the explicit check, not the explicit version.)

Reply:
@grafnu I removed this check from - you might have missed it when you did the re-review.

Reply:
sigh -- yes -- I don't find GitHub's interface particularly easy to use!